### PR TITLE
Document logging on stderr instead of files

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

File: cookbooks/logs.rst [hunk 1]
```diff
@@ -61,6 +69,35 @@ includes language errors such as PHP Errors, Warnings, and Notices, as well as
 uncaught exceptions. It also contains your application logs if you log on
 ``stderr``.
 
+Because SymfonyCloud manages this file for you (preventing disks to get filled
+and using very fast local drives instead of slower network disk), we recommend
+that applications always output their log to ``stder``.
```
Comment (nicolas-grekas) [inline cookbooks/logs.rst:74]: stderr

File: cookbooks/logs.rst [hunk 2]
```diff
@@ -61,6 +69,35 @@ includes language errors such as PHP Errors, Warnings, and Notices, as well as
...
+
+.. _monolog-configuration-stderr:
+
+.. note::
+
+   For developers convenience and for historic reasons, Monolog default recipe
```
Comment (nicolas-grekas) [inline cookbooks/logs.rst:80]: historical?

File: cookbooks/logs.rst [hunk 3]
```diff
@@ -61,6 +69,35 @@ includes language errors such as PHP Errors, Warnings, and Notices, as well as
...
+                   members: [nested, buffer]
+               nested:
+                   type: stream
+       -            path: "%kernel.logs_dir%/%kernel.environment%.log"
+       +            path: php://stderr
+                   level: debug
+               buffer:
+                   type: buffer
+
+   **If you log depreciations, don’t forget to log them on** ``stderr`` **as
```
Comment (nicolas-grekas) [inline cookbooks/logs.rst:98]: deprecations

File: cookbooks/logs.rst (hunk 2 above)
Comment (tucksaun) [reply cookbooks/logs.rst:80]: that's what I wrote initially but then I read again Fabien's blogpost (https://symfony.com/blog/logging-in-symfony-and-the-cloud) where he used "historic" 🤷

File: cookbooks/logs.rst (hunk 3 above)
Comment (tucksaun) [reply cookbooks/logs.rst:98]: why, you don't like Frenglish? :p

File: cookbooks/logs.rst (hunk 2 above)
Comment (fabpot) [reply cookbooks/logs.rst:80]: that's a typo :) historical reasons is the right way.

File: cookbooks/go_live.rst [hunk 4]
```diff
@@ -318,3 +319,31 @@ CDN
...
+Logging
+~~~~~~~
+
+As many other Cloud providers or modern orchestration systems, SymfonyCloud
+recommends logging on ``stderr``. Doing so allows SymfonyCloud to take care for
+you of where and how your logs are stored (read our doc about :doc:`logs` for
+more information).
+
+For Symfony, this is the default behavior of the Kernel since the 3.4 version,
+but for developers convenience, for historic reasons, and probably
```
Comment (fabpot) [inline cookbooks/go_live.rst:332]: historical ;)

Comment (tucksaun) [reply cookbooks/go_live.rst:332]: 👍 I will update the blogpost

File: cookbooks/logs.rst [hunk 5]
```diff
@@ -61,6 +69,35 @@ includes language errors such as PHP Errors, Warnings, and Notices, as well as
...
+                   members: [nested, buffer]
+               nested:
+                   type: stream
+       -            path: "%kernel.logs_dir%/%kernel.environment%.log"
+       +            path: php://stderr
+                   level: debug
+               buffer:
+                   type: buffer
+
+   **If you log deprec`ations, don’t forget to log them on** ``stderr`` **as
```
Comment (fabpot) [inline cookbooks/logs.rst:98]: extra `

**Pull request merged**